elao.app/.editorconfig.dist

```diff
@@ -0,0 +1,54 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 4
+
+################
+# Common files #
+################
+
+[Makefile]
+indent_style = tab
+
+[*.php]
+indent_size = 4
+max_line_length = 120
+
+[*.{css,sass,scss}]
+indent_size = 2
+max_line_length = 120
+
+[*.js]
+indent_size = 2
+max_line_length = 120
+
+[*.{html,xml}]
+indent_size = 4
+
+[*.json]
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false # keep end of line double-space for markdown EOL
+max_line_length = 120 # wrap after X chars
+
+[*.{yaml,yml}]
+indent_size = 2
+
+###########
+# Symfony #
+###########
+
+[*.twig]
```

Comment:
```suggestion
[*.html.twig]
```

?

Sometimes we might generate something else than HTML, and I think the generated format indent matters more than Twig indent. We use 4 for twig in most cases because we use 4 four HTML, right?